.github/workflows/release-service-apps-proxy.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     tags:
       - 'apps-proxy-v**'
+      - 'test-apps-proxy-v**'
```

Comment:
We cannot use it due to suffix matching in azure pipeline. We have to use prefix one. But I susspect that this will be completelly reworked differently as it is now in future.
![image](https://github.com/keboola/keboola-as-code/assets/17671196/f6615426-bfaf-4128-9869-ba16a7efc9b9)